framework/generated/generated_dx12_command_list_util.cpp

```diff
@@ -333,13 +333,9 @@ void Track_ID3D12GraphicsCommandList_OMSetRenderTargets(ID3D12CommandList_Wrappe
     GFXRECON_ASSERT(info != nullptr);
     if(pRenderTargetDescriptors != nullptr)
     {
-        if( NumRenderTargetDescriptors > 0 )
+        for (UINT i = 0; i < (NumRenderTargetDescriptors ? (RTsSingleHandleToDescriptorRange ? 1 : NumRenderTargetDescriptors) : 0); ++i)
```

Comment:
By changing the `param_length` in `dx12_base_generator.py` for `ID3D12GraphicsCommandList_OMSetRenderTargets`/`pRenderTargetDescriptors`, the lookup in https://github.com/andrew-lunarg/gfxreconstruct/blob/eba1b72137e7940f96039abf7b26e3e12300a0f7/framework/generated/dx12_generators/dx12_command_list_util_body_generator.py#L138 no longer succeeds, which is why this generated code changes from an if-statement and a loop inside the body to just a loop.   To my eyes this seems fine, but since this removes any use of `SPECIAL_ARRAY_LENGTH` you could probably remove `SPECIAL_ARRAY_LENGTH` and any processing of `parameter_check_zero` in `add_loop_code()` in `framework/generated/dx12_generators/dx12_command_list_util_body_generator.py`.